tests/test_qaoa.py

```diff
@@ -660,10 +671,29 @@ def circuit(params, **kwargs):
             qml.layer(qaoa_layer, 2, params[0], params[1])
 
         # Defines the device and the QAOA cost function
-        dev = qml.device('default.qubit', wires=len(wires))
+        dev = qml.device("default.qubit", wires=len(wires))
         cost_function = qml.ExpvalCost(circuit, cost_h, dev)
 
         res = cost_function([[1, 1], [1, 1]])
         expected = -1.8260274380964299
 
         assert np.allclose(res, expected, atol=tol, rtol=0)
+
+
+class TestCycles:
+
+    """Test `cycles` functions are behaving correctly"""
```

Comment:
```suggestion
    """Tests that ``cycle`` module functions are behaving correctly"""
```
actually the double backticks don't matter here, but would be needed in standard docstrings.